EIPS/eip-9174.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: 9174
+title: On-chain upgrade signaling
+description: Allows participants to indicate readiness for a client upgrade when producing blocks
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-on-chain-upgrade-signaling/22306
```

Comment:
Added assigned number to Eth Magicians: https://ethereum-magicians.org/t/eip-7848-on-chain-upgrade-signaling/22306